configs/eslint-config-typescript/index.js

```diff
@@ -5,58 +5,62 @@ const ERROR = 2;
 module.exports = {
   root: true,
   extends: [
-    '@monkvision/eslint-config-base',
-    'plugin:@typescript-eslint/recommended',
-    'plugin:import/typescript',
+    "@monkvision/eslint-config-base",
```

Comment:
Your IDE probably modified the punctuation on this file by mistake. Go back to single quotes on the file.